pyproject.toml

```diff
@@ -92,9 +92,13 @@ lint = [
     "tomli",  # for mypy (Python<=3.10)
     "pytest>=6.0",
 ]
+dev = [
```

Comment:
Maybe add a comment, that this extra is intended for use by users of sphinx.util.testing